Makefile

```diff
@@ -119,3 +119,13 @@ update-dep:
 .PHONY: update-otel
 update-otel:
 	$(MAKE) update-dep MODULE=go.opentelemetry.io/collector VERSION=master
+
+.PHONY: otel-from-tree
+otel-from-tree:
```

Comment:
It would be nice to document how to use this. From PR description it sounds useful but I am not sure how to do it.

Reply:
I've added more docs to the makefile (or is there a better place to put it?)